windows/src/desktop/kmshell/xml/config.js

```diff
@@ -130,15 +132,29 @@ document.addEventListener("DOMContentLoaded", windowResize);
       break;
     case 37:    // left
     case 109:   // minus
-      var k = document.getElementById('list_detail_'+n);
-      if( !k ) break;
-      if(k.style.display == 'block') list_detail(event,n);
+      {
+        let keyboard_name = n.substring(9); // remove keyboard_ from element name
+        let kbd_grid = document.getElementById('keyboard_grid_'+keyboard_name);
+        if( !kbd_grid ) break;
+        let kbd_list = document.getElementById('list_'+n);
+        if( !kbd_list ) break;
+        if(!kbd_list.classList.contains('expanded')) break;
```

Comment:
Just wondering why these checks are necessary?

Reply:
I will but the kbd_grid id lower so it is checked just before use. 
First it checks to see if it is a keyboard list (item) and if it is expanded. Without the check you can in through select a keyboard with the mouse, press left arrow, set focus to another keyboard on the list then when, again focusing and expanding the original keyboard the more detail is already show even though the list has never been expanded.
Then show/hide more detail -  `more_detail` blindly does a toggle hence the need to check if it is already in the desired state.